tutorials/scripting/gdscript/gdscript_styleguide.rst

```diff
@@ -46,7 +45,6 @@ Here is a complete class example based on these guidelines:
         set = set_state
     @onready var _state_name = _state.name
 
-
```

Comment:
Same here

Reply:
I double-checked and `black` *does* apply two blank lines before the first method, so this is the correct approach.